### ISSUE TITLE
C18 Coveted Jewel doesn't work properly

### ISSUE THREAD

**Issue by mfalcier** [magefree/mage#5268]
I've attacked a player having a Coveted Jewel on the field with an unblockable creature, and the effect of the jewel triggered. But after that, I haven't received the 3 cards draw or the jewel itself.

**Comment by Kerrus** [comment]
yeah, it untaps the jewel and the owner gets the cards instead of how it's supposed to work.
